internal/server/access_keys_test.go

```diff
@@ -205,6 +205,28 @@ func TestAPI_ListAccessKeys(t *testing.T) {
 		}
 	})
 
+	t.Run("success w/ pagination", func(t *testing.T) {
+		resp := httptest.NewRecorder()
+		req := httptest.NewRequest(http.MethodGet, "/api/access-keys?limit=1", nil)
+		req.Header.Set("Authorization", "Bearer "+adminAccessKey(srv))
+		req.Header.Set("Infra-Version", "0.13.0") // use older version of the API
+
+		routes.ServeHTTP(resp, req)
+		assert.Equal(t, resp.Code, http.StatusOK)
+
+		keys := api.ListResponse[api.AccessKey]{}
+		err = json.Unmarshal(resp.Body.Bytes(), &keys)
+		assert.NilError(t, err)
+
+		// TODO: replace this with a more strict assertion using DeepEqual
+		assert.Equal(t, keys.PaginationResponse, api.PaginationResponse{Limit: 1, Page: 1, TotalCount: 2, TotalPages: 2})
```

Comment:
You can this with cmp.Options as seen in this test file:
https://github.com/infrahq/infra/blob/main/internal/server/data/access_key_test.go

Daniel may have a better suggestion though.

Reply:
I did expand on this, but didn't use cmp.Options{}.